feature/savings/src/commonMain/kotlin/org/mifos/mobile/feature/savings/savingsAccountTransaction/SavingAccountsTransactionScreen.kt

```diff
@@ -122,12 +113,15 @@ internal fun SavingsAccountTransactionScreen(
                     is SavingsAccountTransactionUiState.Success -> {
                         if (uiState.savingAccountsTransactionList.isNullOrEmpty()) {
                             EmptyDataView(
```

Comment:
Make another state (SavingsAccountTransactionUiState.Empty) for your empty data and inside it use EmptyDataView and then inside the `SavingsAccountTransactionUiState.Success` just use the `SavingsAccountTransactionContent`

Have a look in my PR how i did it:
https://github.com/openMF/mifos-mobile/pull/2775